lib/sentry/transport.ex

```diff
@@ -125,7 +125,7 @@ defmodule Sentry.Transport do
   defp client_post_and_validate_return_value(client, endpoint, headers, body) do
     case client.post(endpoint, headers, body) do
       {:ok, status, resp_headers, resp_body}
-      when is_integer(status) and status in 200..599 and is_list(resp_headers) and
```

Comment:
I removed this because it's highly Hackney-specific (Req response readers are a Map).